app/components/Conference/index.scss

```diff
@@ -0,0 +1,340 @@
+        display: inline-block;
+        margin: 0px;
+        padding: 3px;
+      }
+      p.block-bg {
+        font-weight: 700;
+        font-size: 14px;
+        padding: 3px 10px;
+      }
+      h1 {
+        font-size: 30px;
+        display: inline-block;
+        margin: 0px;
+        padding: 3px;
+      }
+      h2.block-bg {
+        font-size: 40px;
+        font-weight: 200;
+      }
+      a {
+        color: $grey;
+      }
+      .sign-up-trampoline {
+        display: -webkit-flex;
+        display: flex;
+        -webkit-flex-flow: column wrap;
+        flex-flow: column wrap;
+        -webkit-align-items: center;
+        align-items: center;
+        img, object {
+          margin-top: 5px;
+          width: 18px;
+          height: 12px;
+        }
+      }
+    }
+
+    @media (min-width: $desktop-width) {
+      #hero {
+        min-height: 660px;
+        background: url("/img/SVG/ReactLondon_SaveTheDate_Web_background-02.svg")
+          no-repeat
+          center center;
+        background-size: cover;
+        .content {
+          margin-top: 300px;
+        }
+        p.block-bg {
+          font-size: 21px;
+          margin-top: 15px;
+        }
+        h1.block-bg {
+          font-size: 45px;
+          margin-bottom: 26px;
+        }
+        h2.block-bg {
+          font-size: 60px;
+          line-height: 1;
+        }
+        .sign-up-trampoline {
+          margin-top: 0px;
+        }
+      }
+    }
+
+    @media (min-width: $mobile-width) and (max-width: $desktop-width) {
+      #hero {
+        p.block-bg {
+          margin-top: 5px;
+        }
+        h2.block-bg {
+          line-height: 1;
+        }
+        .sign-up-trampoline {
+          margin-top: 5px;
+        }
+      }
+    }
+
+    @media (max-width: $mobile-width) {
+      #hero {
+        background: url("/img/SVG/ReactLondon_SaveTheDate_Web_background-03.svg")
+          no-repeat
+          center -135px;
+        background-size: cover;
+        padding-bottom: 20px;
+        .content {
+          margin-top: 178px;
+        }
+        h2.block-bg {
+          line-height: 1;
+        }
+        p.block-bg {
+          margin-top: 5px;
+        }
+        .sign-up-trampoline {
+          margin-top: 5px;
+        }
+      }
+    }
+
+    @media (max-width: $tiny-mobile-width) {
+      #hero {
+        background: url("/img/SVG/ReactLondon_SaveTheDate_Web_background-03.svg")
+          no-repeat
+          center -110px;
+        background-size: cover;
+        .content {
+          margin-top: 142px;
+        }
+        p.block-bg {
+          margin-top: 8px;
+        }
+        .sign-up-trampoline {
+          margin-top: 4px;
+        }
+        h1 {
+          line-height: 1.2;
+        }
+      }
+    }
+
+  // ABOUT
+    #about {
+      background: $conference-color;
+      border-top: none;
+      margin: 22px 0px;
+
+      h2, p {
+        font-size: 16px;
+        @media (min-width: 768px) {
```

Comment:
Changed all except for
@media (max-width: 425px) { // 300px + 125px. An input box smaller than 300px is too small.
because it's only relevant to this section